spec/emarsys/data_objects/email_spec.rb

```diff
@@ -83,6 +83,16 @@
     end
   end
 
+  describe ".unsubscribe" do
+    it "requests an email test sending with custom recipient list" do
```

Comment:
Is this copy-pasta?

Reply:
omg! yes! i'm guilty :(

![giphy-downsized 29](https://user-images.githubusercontent.com/72447/34253348-6ed0a964-e647-11e7-8c18-bb2917713272.gif)